src/vmm/src/logger/metrics.rs

```diff
@@ -779,6 +779,8 @@ pub struct VcpuMetrics {
     pub exit_mmio_write: SharedIncMetric,
     /// Number of errors during this VCPU's run.
     pub failures: SharedIncMetric,
+    /// Number of times that the `KVM_KVMCLOCK_CTRL` ioctl failed.
```

Comment:
My parser doesn't approve the commit message: "but we still it doesn't hurt tracking this."

Reply:
I agree with your parser. (I think) I fixed it.